index.html

```diff
@@ -46,92 +46,8 @@ <h1>Abdel-Khafid SALAOU</h1>
-        <div class="project-card-info deskno">
-            <h3>
-                Hydraulic head loss calculator online
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
-    <div class="project-card three">
-        <a href="#"><img class="cards-img" src="./assets/projects/project_3.svg" alt="My third project"></a>
-        <div class="project-card-info deskno">
-            <h3>
-                Online scientific calculator
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
-    <div class="project-card four">
-        <a href="#"><img class="cards-img" src="./assets/projects/project_4.svg" alt="My fourth project"></a>
-        <div class="project-card-info deskno">
-            <h3>
-                Professional Portfolio
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
-    <div class="project-card deskreduce">
-        <a href="#"><img class="cards-img" src="./assets/projects/project_5.svg" alt="My fifth project"></a>
-        <div class="project-card-info">
-            <h3>
-                Currency converter
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
-    <div class="project-card deskno">
-        <a href="#"><img class="cards-img" src="./assets/projects/project_6.svg" alt="My sixth project"></a>
-        <div class="project-card-info">
-            <h3>
-                Visit counter
-            </h3>
-                <ul class="tag-list">
-                    <li class="tags">HTML/CSS</li>
-                    <li class="tags">Ruby on rails</li>
-                    <li class="tags">Javascript</li>
-                </ul>
-                <button class="see-proj-btn">See this project &rArr;</button>
-        </div>
-    </div>
    </section>
+   <section class="pop-window"></section>
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, kindly ensure your design follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins). For your design kindly make sure the elements on both popups are well aligned just like the way they appear in the given design.
- Your design
![align desktop](https://user-images.githubusercontent.com/63917725/224487467-60ffd95b-4143-4777-9e1f-0a01d7e9f4eb.PNG).
- Given design
![given desktop](https://user-images.githubusercontent.com/63917725/224487827-2428be12-3e70-4e87-859d-7b982a05d346.PNG)

- [ ] Kindly make sure your mobile popup also follows the given design. You may align the elements correctly and also fix this dark button as well as the paragraph. Kindly align it properly to make your design a close replica of the design given.
- Your design
![font and button](https://user-images.githubusercontent.com/63917725/224487744-971f0c3c-6d60-4af2-b2af-823db8153612.PNG)
- Given design
![given mobile](https://user-images.githubusercontent.com/63917725/224487779-9b274d80-a4c3-4c11-a6f3-098915481655.PNG).

- [ ] [Optional] Kindly apply a blur to your background whenever a popup is displayed. This will create a more visible design

Reply:
Okay @1GORDON, thank you for your time. Fixing this right away.

Reply:
![image](https://user-images.githubusercontent.com/119133987/224493735-067add95-dcbe-43aa-8718-01890a3cab2f.png)
template